pkg/kapp/resources/identified_resources_list.go

```diff
@@ -11,10 +11,14 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-func (r IdentifiedResources) List(labelSelector labels.Selector, resRefs []ResourceRef) ([]Resource, error) {
+type IdentifiedResourcesListOpts struct {
+	UseCached bool
```

Comment:
```suggestion
	UseCachedResTypes bool
```
We could be more specific

Reply:
Used IgnoreCachedResTypes